src/components/Footer/footer.module.scss

```diff
@@ -0,0 +1,137 @@
+@use './../../scss/colors';
+
+.footerDiv {
+  position: relative;
+  margin-top: 2rem;
+  background-image: url("./../../assets/images/footer/origami.webp");
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+
+  .topShape {
+    overflow: hidden;
+    position: relative;
+    top: 0;
+    left: 0;
+    z-index: 1;
+    width: 100%;
+    line-height: 0;
+    direction: ltr;
+
+    svg {
+      display: block;
+      width: calc(100% + 1.3px);
+      position: relative;
+      left: 50%;
+      -webkit-transform: translateX(-50%) rotateY(180deg);
+      -ms-transform: translateX(-50%) rotateY(180deg);
+      transform: translateX(-50%) rotateY(180deg);
+      overflow: hidden;
+
+      .svgPath {
+        fill: colors.$white;
+        -webkit-transform-origin: center;
+        -ms-transform-origin: center;
+        transform-origin: center;
+        -webkit-transform: rotateY(0deg);
+        transform: rotateY(0deg);
+      }
+    }
+  }
+  .tint {
+    position: absolute;
+    top: 0;
+    left: 0;
+    height: 100%;
+    width: 100%;
+    background: colors.$black_0-8;
+  }
+
+  .newsletter {
+    position: relative;
+    color: colors.$white;
+    text-align: center;
+    padding: 50px 0 70px 0;
+
+    .heading {
+      h5 {
+        font-size: 2.5rem;
+        font-weight: 300;
+        margin-bottom: 3rem;
+      }
+      h6 {
+        font-weight: 300;
+        margin-bottom: 0;
+      }
+    }
+    
+    .emailField {
+      min-width: 320px;
+      text-align: center;
+      margin: 0 5px;
+
+      .form {
+        display: inline-block;
+        display: flex;
+        align-items: stretch;
+        justify-content: center;
+
+        .box {
+          height: 40px;
+          background-color: colors.$white;
+          border-radius: 30px 0 0 30px;
+          display: flex;
+          align-items: center;
+          padding: 15px;
+          min-width: 220px;
+
+          .icon {
+            padding: 10px;
+            color: colors.$blue1;
+            min-width: 40px;
+          }
+          input {
+            flex: 1;
+            height: 30px;
+            border: none;
+            outline: none;
+            font-size: 16px;
+          }
+        }
+        .bsClass {
+          border-radius: 0 30px 30px 0;
+          padding: 6px 15px;
+          background-color: colors.$blue1;
```

Comment:
give proper names to these like primary blue or light blue that would be easy for someone to understand.